Sources/Playbook/Components/Progress Simple/PBProgressSimple.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Playbook Swift Design System
+//
+//  Copyright © 2024 Power Home Remodeling Group
+//  This software is distributed under the ISC License
+//
+//  PBProgressSimple.swift
+//
+
+import SwiftUI
+
+public struct PBProgressSimple: View {
+  @Binding var progress: Double
+  @Binding var value: Int
+  let maxValue: Int
+  let progressColor: Color
+  let variant: Variant
+  
+  public init(
+    progress: Binding<Double> = .constant(0.0),
+    value: Binding<Int> = .constant(2),
+    maxValue: Int = 10,
+    progressColor: Color = .pbPrimary,
+    variant: Variant = .default
+    
+  ) {
+    self._progress = progress
+    self._value = value
+    self.maxValue = maxValue
+    self.progressColor = progressColor
+    self.variant = variant
+  }
+  
+  public var body: some View {
+    variantView
+  }
+}
+
+public extension PBProgressSimple {
+  enum Variant {
+    case `default`, settingValue
+  }
+  
+  var variantView: some View {
+    HStack {
+      switch variant {
+      case .default: ProgressView(value: progress, total: 1)
```

Comment:
Why is 1 the total value? Is it right?

Reply:
Yes, it's a double value so 1 is essentially "up to" 100%.

Reply:
idk if this makes sense but maybe another way to say is that its calculating the progress value by what the total allowed is. so if this tracks up to 100% then the value provided in the catalog will be incremented based on the total.